examples/restore_login.py

```diff
@@ -1,71 +1,90 @@
+#!/usr/bin/env python3
+
 import asyncio
 import json
 import os
 import sys
+import getpass
 
 from nio import AsyncClient, LoginResponse
 
 CONFIG_FILE = "restore_login_config.json"
 
 # Check out main() below to see how it's done.
 
-def write_details_to_disk(resp: LoginResponse) -> None:
+
+def write_details_to_disk(resp: LoginResponse, home_server) -> None:
     """Writes the required login details to disk so we can log in later without
     using a password.
 
     Arguments:
         resp {LoginResponse} -- the successful client login response.
+        home_server -- the name of the homeserver, e.g. "https://matrix.example.org"
```

Comment:
`homeserver` is better described as the URL of the homeserver, not the name. You can reuse the description from AsyncClient: "The URL of the homeserver which we want to connect to."

Reply:
Good suggestion!

commit 2 will have this rephrased as URL as you suggested.